spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -113,86 +102,119 @@ function ExecuteTx(sourcePort: string, destPort: string, destChannel: string, ms
 
+Combined with the one channel per interchain account approach, this method of version negotiation allows us to pass the address of the interchain account back to the controller chain and create a mapping from controller port ID -> interchain account address during the `OnChanOpenAck` callback. As outlined in the [controlling flow](#Controlling-Flow), a controller chain will need to know the address of a registered interchain account in order to send transactions to the account on the host chain.
 
-**Controlling Flow**
 
-Once an interchain account is registered on the host chain a controller chain can begin sending instructions (messages) to control this account. 
+#### Version negotiation summary
 
-1. The controller chain calls `GetInterchainAccountAddressFromAck()` to get the address of the interchain account on the host chain and sets the address in state mapped to the respective portID. 
-2. The controller chain calls `TrySendTx` and passes message(s) that will be executed on the host side by the associated interchain account (determined by the source port identifer)
+| Initiator | Datagram         | Chain acted upon |  Version (Controller, Host) | Port ID (Controller, Host) |
+| --------- | ---------------- | ---------------- |  ---------------------- | ---------------------- |
+| Controller| ChanOpenInit     | Controller       |  (ics27-1, none)        | (ics27-1.0.0.<owner-id>, interchain-account)           |
+| Relayer   | ChanOpenTry      | Host             |  (ics27-1, ics27-1.<owner-id>)        | (ics27-1.0.0.<owner-id>, interchain-account)       |
+| Relayer   | ChanOpenAck      | Controller       |  (ics27-1, ics27-1.<owner-id>)        | (ics27-1.0.0.<owner-id>, interchain-account)        | (ics27-1.0.0.<owner-id>, interchain-account)        |
+| Relayer   | ChanOpenConfirm  | Host             |  (ics27-1, ics27-1.<owner-id>)        | (ics27-1.0.0.<owner-id>, interchain-account)           | (ics27-1.0.0.<owner-id>, interchain-account)           |
+
+
+
+#### Controlling Flow
+
+Once an interchain account is registered on the host chain a controller chain can begin sending instructions (messages) to the host chain to control the account. 
+
+1. The controller chain calls `TrySendTx` and passes message(s) that will be executed on the host side by the associated interchain account (determined by the controller side port identifier)
 
 Cosmos SDK psuedo code example:
 
 ```typescript
 interchainAccountAddress := GetInterchainAccountAddress(portId)
 msg := &banktypes.MsgSend{FromAddress: interchainAccountAddress, ToAddress: ToAddress, Amount: amount}
+icaPacketData = InterchainAccountPacketData{
+   Type: types.EXECUTE_TX,
+   Data: serialize(msg),
+   Memo: "memo",
+}
+
 // Sends the message to the host chain, where it will eventually be executed 
-TrySendTx(ownerAddress, connectionId, counterPartyConnectionId, msg)
+TrySendTx(chanCapability, ownerAddress, connectionId, counterPartyConnectionId, data)
```

Comment:
```suggestion
TrySendTx(ownerAddress, connectionId, counterPartyConnectionId, data)
```

The capabilities aren't mentioned in the spec since it is SDK specific authentication schema. Similar to how we don't include `ctx sdk.Context` in our spec documents.

I would just say that it is responsibility of ICA module to authenticate that the caller of `TrySendTx` is the owner of the interchain account

Reply:
Cool. Good to know :handshake: